lib/cider.dart

```diff
@@ -1,3 +1,11 @@
-export 'package:cider/src/console/console.dart';
-export 'package:cider/src/console/console_application.dart';
-export 'package:cider/src/console/exit_code.dart';
+library cider;
+
+export 'package:cider/src/console/console.dart'
+    // ignore: uri_does_not_exist
+    if (dart.library.html) 'package:cider/src/web_support_stub.dart';
```

Comment:
Why is this needed? Do we intend to run in a browser?

Reply:
This is for pub pana rating)

Reply:
To get 110 out of 110.

Reply:
I prefer clean an concise code over the pub rating, tbh. Let's optimize towards 100% test coverage instead.